src/styles/_overrides.scss

```diff
@@ -7,3 +7,11 @@
 [class^='FourOhFour-module--link']:before {
   color: $text-inverse !important;
 }
+
+// Header icon style fix needed because the Carbon website is on a
+// newer version of Carbon vs. the Gatsby theme
+// If the Gatsby theme is updated to use latest Carbon, this
+// shouldn't be needed
```

Comment:
It looks like `3.1.2` was just released for `gatsby-theme-carbon` which bumps the `@carbon` version to `1.32.0`. Should we try updating that and see if the issue resolves itself?

Reply:
Just checked, both issues are still there 😓